qcelemental/molparse/to_string.py

```diff
@@ -136,6 +138,24 @@ def to_dict(self) -> Dict:
 
         smol.extend(atoms)
 
+    if dtype == "orca":
+        # Notes
+        # * if units not in umap (e.g., nm), can't be read back in by from_string()
+
+        atom_format = "{elem}" if atom_format is None else atom_format
+        ghost_format = "@{elem}" if ghost_format is None else ghost_format
+        umap = {"bohr": "au", "angstrom": ""}
+
+        atoms = _atoms_formatter(molrec, geom, atom_format, ghost_format, width, prec, 2)
+        nat = len(atoms)
+
+        first_line = """"""
+        smol = [first_line.rstrip()]
+        smol.append(f"{'*xyz '} {int(molrec['molecular_charge'])} {molrec['molecular_multiplicity']}")
+
+        smol.extend(atoms)
+        smol.append("*")
+
```

Comment:
good to get some tests here (https://github.com/MolSSI/QCElemental/blob/master/qcelemental/tests/test_molparse_to_string.py#L188)